### DISCUSSION TITLE
Should we allow to change at least every place having the default theme color in the opinionated theme builder?

### DISCUSSION THREAD

**Discussion by kukoc** [dxc-technology/halstack-react#1444] [General]
I run into this issue recreating the Bloom theme. It's based mainly on 2 colors, blue and green. Some components will be displaying the purple color no matter what you configure in Ux config or opinionated theme builder. This is the case with the hover color of the inputs, which is only purple. 

Should we allow this basic configuration to at least replace this main color (purple) with another theme's main color?

Example of hovered input that can't be configured:

![image](https://user-images.githubusercontent.com/7778817/213166874-0a4b22bd-b1cf-4bc9-8150-1eb20f08cbd9.png)

**Comment by raquelarrojo** [comment]
I agree, and as designers also, I think this hover color should be also customised using the default theme.
We realised about it when creating a new theme, so I think it is something needed.

**Reply by rconcepcion3** [reply]
Yup, I agree with this... some items to consider in terms of design tokens across components where this might be relevant:

hover color (hoverBorderColor)

Links
visited and visited underline colors (visitedFontColor/visitedUnderlineColor)

Tabs
Hover, Visited, Background, Text, Icon, and border colors
